sample_data_from_verso/data/profiles/BIBFRAME 2.0 Title Information.json

```diff
@@ -400,119 +400,6 @@
                 "resourceURI": "http://id.loc.gov/ontologies/bibframe/CollectiveTitle",
                 "resourceLabel": "Collective Title"
             },
-            {
```

Comment:
I'm dubious that all these title flavors would have just disappeared with no replacements ...

Reply:
The various variant titles were put in their own RTs
See http://bibframe.org/bfe/index.html, Create Resource-> Notated Music -> Create Instance -> "Collective Title"